go.mod

```diff
@@ -60,6 +59,7 @@ require (
 	github.com/leodido/go-urn v1.2.1 // indirect
 	github.com/shurcooL/graphql v0.0.0-20220606043923-3cf50f8a0a29 // indirect
 	github.com/sirupsen/logrus v1.9.0 // indirect
+	github.com/yourbasic/graph v0.0.0-20210606180040-8ecfec1c2869
```

Comment:
This is a 5 year old dependency. Is there a way to avoid using this?

https://github.com/yourbasic/graph